tests/test_split_flatpak_repo.py

```diff
@@ -471,11 +560,11 @@ def test_duplicate_repo(self):
         for root, dirs, files in os.walk(dup_path):
             repo_dir = os.path.relpath(root, dup_path).split(os.sep)[0]
             if repo_dir in ('objects', 'deltas'):
-                expected_links = 2
+                test = self.assertGreater
```

Comment:
In what cases are the expected_links greater than 2? Is it roughly, if part of a Flatpak, then 3 (1 in system repo, 1 in flatpak repo, 1 deployed) and otherwise, 2?

Reply:
Exactly. I thought about trying to keep this exact, but really I just want to make sure some things are hard linked and others aren't. I don't think the exact count matters that much.